migrations/challenges/sync-all-challenges.js

```diff
@@ -0,0 +1,40 @@
+import Bluebird from 'Bluebird';
+
+import { model as Challenges } from '../../website/server/models/challenge';
+import { model as User } from '../../website/server/models/user';
+
+async function syncChallengeToMembers (challenges) {
+  challenges.forEach(async function (challenge) {

```

Comment:
well the problem is that any error inside these async functions will get swallowed as they're not part of a chain with a `catch` statement or must be wrapped inside a try/catch `async function () { try {code} catch (e) {handle}` and also that some challenges are really big and the database would crash. Consider that the onboarding emails process only 10 users in parallel and the cpu on the database grows to more than 20%